src/tracks/SampleTrack.cpp

```diff
@@ -652,9 +1088,21 @@ void SampleTCOView::paintEvent( QPaintEvent * pe )
 	p.end();
 
 	painter.drawPixmap( 0, 0, m_paintPixmap );
+	drawFades();
 }
 
 
+void SampleTCOView::enterEvent(QEvent *e)
+{
+	if (e != nullptr) { m_inside = true; }
```

Comment:
Can `e` be a `nullptr`? I guess if called manually, but I don't think it will ever be `nullptr` when Qt generates it.

Reply:
Probably not. I think it is good to check anyway, to be on the safe side.

Reply:
Well the reason I bring it up is because the event is called only if raised, so `e` will never be `nullptr`. The only time it will be is if we were to call it manually without.